astroquery/linelists/cdms/core.py

```diff
@@ -390,9 +394,28 @@ def find(self, st, flags):
 def build_lookup():
 
     result = CDMS.get_species_table()
-    keys = list(result[1][:])  # convert NAME column to list
-    values = list(result[0][:])  # convert TAG column to list
+    keys = list(result['molecule'][:])  # convert NAME column to list
+    values = list(result['tag'][:])  # convert TAG column to list
     dictionary = dict(zip(keys, values))  # make k,v dictionary
     lookuptable = Lookuptable(dictionary)  # apply the class above
 
     return lookuptable
+
+
+def retrieve_catfile(url='https://cdms.astro.uni-koeln.de/classic/entries/partition_function.html'):
+    """
+    Simple retrieve index function
+    """
+    import requests
```

Comment:
do the import on the top, we only do this type when it's about an optional dependency (and in practice `requests` is very likely imported already, so there is no real overhead doing it at the top)